### PR TITLE
Gcesana adding clouds

### PR DESCRIPTION
**Pull request by gcesana**
Just filling in

### CODE REVIEW

**Review by gcesana — COMMENTED** [review]
Definitely the best commit I've ever seen

File: gcesana-adding-clouds.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+ # Pasadena, CA
+
+ ### Best burger in town
+     The counter
```
Comment (fkuykend) [inline gcesana-adding-clouds.md]: counter --> Counter

**Review by fkuykend — CHANGES_REQUESTED** [review]

File: gcesana-adding-clouds.md (hunk 1 above)
Comment (gcesana) [reply gcesana-adding-clouds.md]: Thanks!
If you have suggestion for another great burger, please shoot 🍔

**Comment by gcesana** [comment]
@fkuykend What about now?

**Pull request merged**